### PR TITLE
Handle gas station API failures more gracefully

### LINKED ISSUE #1663: Handle when Etherchain gas-station API is down
We switched to [etherchain](https://www.etherchain.org/api/gasPriceOracle) in #1641 from ethgasstation b/c the latter requires an API key beginning July 1. The former API as of now appears down. 

Additionally:  every time the API call fails, the GasEstimator bot sends a warning which spams our alert channels. 

Possible solutions:
- Add other gas station API's as backup sources, such as [etherscan (requires an API key)](https://etherscan.io/apis) and ethgasstation
- Include gas station API key in bot configs
- Reduce this specific log level to `info` or `debug` if we just fallback harmlessly to the default gas price so we don't get spammed on PagerDuty and/or Slack

### PR DESCRIPTION
**Pull request by nicholaspai**
Changelog:
- Change Logger level for GasEstimator errors from `error` to `debug` so that if an external API fails (like Etherchain did recently), then it won't keep sending Pagerduty/Slack alerts.
- Add a backup API, etherscan, to fall back to before falling back on default value.
- Update default gas price to be conservatively higher, in line with current safe gas prices of ~50

Questions for @chrismaree @mrice32 :
- Should I make the logs level `debug` or `info`? Note that `info` would print to Slack (but not PagerDuty)

Resolves #1663

### CODE REVIEW

File: financial-templates-lib/helpers/GasEstimator.js [hunk 1]
```diff
@@ -65,15 +67,33 @@ class GasEstimator {
...
+        message: "client polling error, trying backup API🚨",
         error: typeof error === "string" ? new Error(error) : error
       });
 
+      // Try backup API.
+      try {
+        const responseBackup = await fetch(backupUrl);
+        const jsonBackup = await responseBackup.json();
+        if (jsonBackup.SafeGasPrice) {
+          return jsonBackup.fast;
```
Comment (mrice32) [inline financial-templates-lib/helpers/GasEstimator.js:84]: Can you document the structure of this JSON? Is it possible for `SafeGasPrice` to exist but `fast` to not exist? Just want to make sure we don't return `undefined`.

Comment (mrice32) [reply financial-templates-lib/helpers/GasEstimator.js:84]: Sorry, noticed this after my initial approval.

Comment (nicholaspai) [reply financial-templates-lib/helpers/GasEstimator.js:84]: Ah this looks like a typo it should read `jsonBackup.SafeGasPrice`

Comment (mrice32) [reply financial-templates-lib/helpers/GasEstimator.js:84]: On an unrelated note, do they just have safe? Or do they have fast too?

Comment (nicholaspai) [reply financial-templates-lib/helpers/GasEstimator.js]: Fixed

Comment (nicholaspai) [reply financial-templates-lib/helpers/GasEstimator.js]: THey have something called `ProposeGasPrice` which appears faster we can use that.  It looks equivalent to the `FastestGasPrice` for etherchain.

**Comment by mrice32** [comment]
> * Should I make the logs level `debug` or `info`? Note that `info` would print to Slack (but not PagerDuty)

I think the way you have it makes sense. If one or both of the APIs are down for a while, we probably don't want it spamming slack.

**Pull request merged**